### PR TITLE
Adding modular flat lists and new api end point

### PR DESCRIPTION
**Pull request by AllanCSR09**
image
image

### CODE REVIEW

File: apps/expo/src/app/userprofile/index.tsx [hunk 1]
```diff
@@ -5,16 +5,50 @@ import { SafeAreaView } from "react-native-safe-area-context";
 import MIcon from "react-native-vector-icons/MaterialCommunityIcons";
 import { LinearGradient } from "expo-linear-gradient";
 
+import HorizontalList from "../../commons/HorizontalList";
 //import { MaterialCommunityIcons } from "react-native-vector-icons";
 import TitleStack from "../../commons/TitleStack";
 
+const defaultArray = [
```
Comment (hernan28) [inline apps/expo/src/app/userprofile/index.tsx:12]: best approach is to set a TODO comment so we can know later on to change the hardcoded values

File: apps/expo/src/app/userprofile/index.tsx [hunk 2]
```diff
@@ -5,16 +5,50 @@ import { SafeAreaView } from "react-native-safe-area-context";
 import MIcon from "react-native-vector-icons/MaterialCommunityIcons";
 import { LinearGradient } from "expo-linear-gradient";
 
+import HorizontalList from "../../commons/HorizontalList";
 //import { MaterialCommunityIcons } from "react-native-vector-icons";
```
Comment (hernan28) [inline apps/expo/src/app/userprofile/index.tsx:9]: You can delete this comment

**Review by hernan28 — APPROVED** [review]

File: apps/expo/src/app/userprofile/index.tsx (hunk 1 above)
Comment (AllanCSR09) [reply apps/expo/src/app/userprofile/index.tsx:12]: Got it! Will do in next PRs

File: apps/expo/src/app/userprofile/index.tsx (hunk 2 above)
Comment (AllanCSR09) [reply apps/expo/src/app/userprofile/index.tsx:9]: Gotcha, doing it right away

**Pull request merged**